### DISCUSSION TITLE
golangci-lint: command not found

### DISCUSSION THREAD

**Discussion by cx-pavel-monin** [golangci/golangci-lint#1898] [Q&A] [answered]
Hello!
I am trying to install golangci-lint and it installs fine. But when I try to use it, I get a message like this:
golangci-lint: command not found
I tried all the options from https://golangci-lint.run/usage/install/ but nothing works.

What did I miss?

**Comment by ldez** [comment]
Hello,

the binary path must be referenced in your `PATH`

**Reply by cx-pavel-monin** [reply]
What is  binary path? 
My PATH is $(go env GOPATH) ?

**Reply by cx-pavel-monin** [reply]
I added it in PATH export 
PATH=$PATH:$(go env GOPATH)/bin/golangci-lint
And I stay in ~/go/bin$ try to use golangci-lint and stell got the same error.

**Reply by bombsimon** [reply]
You shouldn't add the executable itself, only the path to it. So if the file `golangci-lint` is in `$GOPATH/bin`, your `PATH` should be `PATH=$PATH:$(go env GOPATH)/bin`. When that is set up properly, you can invoke the command from any directory.

**Reply by cx-pavel-monin** [reply]
Thank you, kind person!
Wish you health, happiness, and all the best!

**Comment by cx-pavel-monin** [comment, marked as answer]
Set PATH permanently:
`sudo nano ~/.bashrc`
add and save in this file this string: export PATH=$PATH:$(go env GOPATH)/bin/

**Comment by alexbozhenko** [comment]
I think a better file for this is `.profile`
Go installation instructions for linux https://go.dev/doc/install recommend it as well.
Stackoverflow explains: https://serverfault.com/a/261807

So I recommend to do this:
```
echo 'export PATH=$PATH:$(go env GOPATH)/bin' >> ~/.profile
```

**Reply by enosras** [reply]
alexbozhenko has the most accurate answer. it worked for me, when I moved the path from .zshrc to .zshprofile
